src/main/java/Applicaiton.java

```diff
@@ -0,0 +1,214 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import enumeration.Commands;
+
+import static enumeration.Commands.*;
+
+import enumeration.Genre;
+
+import static enumeration.Genre.*;
+
+import models.Song;
+import models.Singer;
+
+/**
+ * The Application class is the main class in the application.
+ */
+public class Applicaiton {
```

Comment:
This class has too much logic in it. Better to put the logic from it in additional class